fibers/posix-clocks-linux.scm

```diff
@@ -1,17 +1,19 @@
-;; POSIX clocks
+;; POSIX clocks (Linux)
```

Comment:
This isn't Linux, this is just POSIX, so I think the original name is correct (or something like fibers/posix-clocks-generic.scm, because of how things are set up).

Reply:
Removed. I've also named the file `posix-clocks-generic.scm`. And the `linux` PLATFORM is now `generic`.